packages/itwinui-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -411,6 +414,14 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
         const newArray = selectedChangeHandler(__originalIndex, actionType);
         dispatch({ type: 'multiselect', value: newArray });
         onChangeHandler(__originalIndex, actionType, newArray);
+
+        // update live region
+        setLiveRegionSelection(
```

Comment:
Would it be usefull to add 'Selected items' for screen reader to announce before announcing a list of items? It might be confusing why it is listing random numbers.

Reply:
maybe. but then it would need to be localized (one more thing devs have to worry about)